packages/editor/src/client/atoms/poll/pollAnswers.tsx

```diff
@@ -0,0 +1,179 @@
+import {ApolloError} from '@apollo/client'
+import PlusIcon from '@rsuite/icons/legacy/Plus'
+import TrashIcon from '@rsuite/icons/legacy/Trash'
+import React, {useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {Button, Col, Form, IconButton, Message, Modal, Row, toaster} from 'rsuite'
+
+import {
+  FullPoll,
+  PollAnswer,
+  PollAnswerWithVoteCount,
+  useCreatePollAnswerMutation,
+  useDeletePollAnswerMutation
+} from '../../api'
+
+interface PollAnswersProps {
+  poll?: FullPoll
+  onPollUpdated(poll: FullPoll): void
+  onAnswerAdded(answer: PollAnswer): Promise<void>
+  onAnswerDeleted(): Promise<void>
+}
+
+export function PollAnswers({
+  poll,
+  onPollUpdated,
+  onAnswerAdded,
+  onAnswerDeleted
+}: PollAnswersProps) {
+  const {t} = useTranslation()
+  const [modalOpen, setModalOpen] = useState<boolean>(false)
+  const [answerToDelete, setAnswerToDelete] = useState<PollAnswerWithVoteCount | undefined>(
+    undefined
+  )
+  const [newAnswer, setNewAnswer] = useState<string>('')
+  const [createAnswerMutation, {loading}] = useCreatePollAnswerMutation()
+  const [deleteAnswerMutation] = useDeletePollAnswerMutation()
+
+  const onErrorToast = (error: ApolloError) => {
+    toaster.push(
+      <Message type="error" showIcon closable duration={3000}>
+        {error.message}
+      </Message>
+    )
+  }
+
+  /**
+   * FUNCTIONS
+   */
+  async function createAnswer() {
+    if (!poll) {
+      return
+    }
+    if (newAnswer === '') {
```

Comment:
```suggestion
    if (!newAnswer) {
```

As we want to check if newAnswer is empty, could lead to a bug and opens up the question when reading the code why only empty string.